css/style.css

```diff
@@ -19,7 +19,6 @@ body{
     height: 217vh;
     width: 90%;
     max-width: 90%;
-    max-height: 217vh;
```

Comment:
To make the page responsive:
- The height should not be set, it should be fluid
- `max-width` should always be a fixed number, setting it as % doesn't make sense.
```
.main{
    margin-top: 35px;
    width: 90%;
    max-width: 1200px;
    background-color: white;
}
```